modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/near/GridNearGetRequest.java

```diff
@@ -499,46 +479,38 @@ public long accessTtl() {
                 reader.incrementState();
 
             case 10:
-                mvccSnapshot = reader.readMessage("mvccSnapshot");
-
-                if (!reader.isLastRead())
-                    return false;
-
-                reader.incrementState();
-
-            case 11:
                 readersFlags = reader.readCollection("readersFlags", MessageCollectionItemType.BOOLEAN);
 
                 if (!reader.isLastRead())
                     return false;
 
                 reader.incrementState();
 
-            case 12:
+            case 11:
                 taskNameHash = reader.readInt("taskNameHash");
 
                 if (!reader.isLastRead())
                     return false;
 
                 reader.incrementState();
 
-            case 13:
+            case 12:
                 topVer = reader.readAffinityTopologyVersion("topVer");
 
                 if (!reader.isLastRead())
                     return false;
 
                 reader.incrementState();
 
-            case 14:
+            case 13:
                 txLbl = reader.readString("txLbl");
 
                 if (!reader.isLastRead())
                     return false;
 
                 reader.incrementState();
 
-            case 15:
+            case 14:
```

Comment:
Field removed, but `fieldsCount` is not changed (for other messages too). This can lead to communication SPI failure.
It's better to use `MessageCodeGenerator` to regenerate serialization/deserialization logic to avoid such errors.